cmd/common/image/build.go

```diff
@@ -88,6 +90,8 @@ func NewBuildCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&opts.local, "local", "l", false, "Build using local tools")
 	cmd.Flags().StringVarP(&opts.image, "tag", "t", "", "Name for the built image (format name:tag)")
 	cmd.Flags().StringVar(&opts.builderImage, "builder-image", DEFAULT_BUILDER_IMAGE, "Builder image to use")
+	cmd.Flags().BoolVar(&opts.updateMetadata, "update-metadata", false, "Update the metadata according to the eBPF code")
+	cmd.Flags().BoolVar(&opts.validateMetadata, "validate-metadata", true, "Validate the metadata file before building the gadget image")
```

Comment:
I would like if the flag names would be all constants (Do this in another PR for all flags)

Reply:
What would be the advantage of having them as constants? We're not using this value in any other place. I actually think it's more clear having the value directly here instead of having to look for the const declaration in another place.